docs/setup/settings.asciidoc

```diff
@@ -249,77 +249,73 @@ To reload the logging settings, send a SIGHUP signal to {kib}.
 [cols="2*<"]
 |===
 
-|[[logging-dest]] `logging.dest:`
- | Enables you to specify a file where {kib} stores log output.
-*Default: `stdout`*
+|[[logging-root]] `logging.root:`
+| The `root` logger has a dedicated configuration node since this context name is special and must be configured for logging. 
```

Comment:
> and must be configured for logging. 

Do you mean we always apply a default config for it? From the reader's POV it looks like *they* have to configure it.

Reply:
I took that phrase directly from the README and agree it's rather vague. AFAIK, we don't automatically configure the default appender for root logger so they will have to do that themselves. I'll change the wording there and we can revisit the requirement if we decide to do that as a default.

Reply:
> we don't automatically configure the default appender for root logger so they will have to do that themselves

I believe we configure the default appender for root logger unless a user specifies something else in `logging.root.appenders`, in which case they will need to also include `default`.

Reply:
@lukeelmers You are absolutely right! I hadn't even tried that before now.